public/styles/main.css

```diff
@@ -399,11 +399,10 @@ div.__shecodecard>div>div>div>p.__shecodevmvp1 {
     padding: 40px;
     border-radius: 5px;
     margin-top: 27%;
-
-
+    height: 27rem;
```

Comment:
@OdunayoOkebunmi Is there something this height is helping us achieve with the equal heights? From the latest changes, it doesn't seem like it is needed anymore, it only adds more space below on desktop view. See below:

Without the height applied:
![screenshot-sca-website-oo-72-updat-hppr4p herokuapp com-2020 03 08-17_07_07](https://user-images.githubusercontent.com/17818475/76166518-77ec8c80-615f-11ea-828e-0b99e9b788ce.png)

With the height applied:
![screenshot-sca-website-oo-72-updat-hppr4p herokuapp com-2020 03 08-17_08_02](https://user-images.githubusercontent.com/17818475/76166531-89ce2f80-615f-11ea-9219-f2f2f7491843.png)

Please check to see if it's still necessary